### ISSUE TITLE
Admin Console?

### ISSUE THREAD

**Issue by skyem123** [Nightfall/MCMUCH-Rust-Website#4]
Would be useful...

**Comment by RX14** [comment]
Yes, this will be super useful. Before this admin will have to be done with an interactive ruby console which will be super easy to fuck up stuff.

**Comment by skyem123** [comment]
This is Rust, it doesn't have an interactive console.

**Comment by RX14** [comment]
I know, but the API - the only part with direct db access - is written in Ruby, so we will have to admin from the ruby interactive console before this admin console is done.

**Comment by skyem123** [comment]
This admin console will only handle the website.

**Comment by RX14** [comment]
Administrating what? The website should have no state.

**Comment by skyem123** [comment]
stuff like reloading and invalidating all cookies.

**Comment by RX14** [comment]
Hmmn, it seems like a solution to a problem that doesn't exist yet. Once a problem appears that you need something like this, then for sure build it. I just wouldn't spend time on this before you think it's neccesary.
